subt/artf_utils.py

```diff
@@ -0,0 +1,13 @@
+from subt.artifacts import (RESCUE_RANDY, BACKPACK, PHONE, HELMET, ROPE, EXTINGUISHER, DRILL, VENT, CUBE)
```

Comment:
I am thinking about moving it all here and instead import it from `artf_utils` (needed for `control_center_qt.py` with colliding cv2 libs)